src/AspNet.Security.OAuth.SuperOffice/SuperOfficeAuthenticationHandler.cs

```diff
@@ -109,7 +122,7 @@ public partial class SuperOfficeAuthenticationHandler : OAuthHandler<SuperOffice
             }
         }
 
-        return contextIdentifier;
+        return await Task.FromResult((contextIdentifier, webApiUrl));
```

Comment:
You should be able to just return this:

```suggestion
        return (contextIdentifier, webApiUrl);
```

Reply:
The webapiurl claim has always been there, this is not what has changed. What has changed is written in the [issue](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/issues/738). 

In short, the subdomain (sub) was *fixed*. Now it can be sub or sub2 or sub3, etc. Because of this change, and because the UserInfo endpoint is an API call rather than IAW the standard, the UserInfo endpoint request can be on one of a few subdomains.

The webapiurl claim is guaranteed to be correct, and therefore validating it doesn't make sense.